gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/UserTaskController.java

```diff
@@ -75,4 +109,20 @@ private static ProblemDetail mapRejectionToProblem(final BrokerRejection rejecti
         }
     };
   }
+
+  private static Optional<ResponseEntity<Object>> validateAssignmentRequest(
+      final UserTaskAssignmentRequest assignmentRequest) {
+    if (assignmentRequest.getAssignee() == null || assignmentRequest.getAssignee().isBlank()) {
+      final String message =
+          String.format(
+              "Command '%s' validation failed with code '%s': %s",
+              UserTaskIntent.ASSIGN, INVALID_ARGUMENT, "No assignee provided");
```

Comment:
🔧 Let's keep it simple, maybe.
```suggestion
      final String message = "No assignee provided";
```